resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+        logo: null,
+        favicon: null,
+        roomLimit: null,
+        paginationPageSize: null,
+        ownRoomsPaginationPageSize: null
+      },
+      errors: {}
+    };
+  },
+  methods: {
+    /**
+     * Handle get settings data
+     */
+    getSettings () {
+      this.isBusy = true;
+      Base.call('settings')
+        .then(response => {
+          this.settings.name = response.data.data.name;
+          this.settings.logo = response.data.data.logo;
+          this.settings.favicon = response.data.data.favicon;
+          this.settings.roomLimit = response.data.data.room_limit;
+          this.settings.ownRoomsPaginationPageSize = response.data.data.own_rooms_pagination_page_size;
+          this.settings.paginationPageSize = response.data.data.pagination_page_size;
+          this.roomLimitMode = (this.settings.roomLimit === -1 ? 'unlimited' : 'custom');
+        })
+        .catch((error) => {
+          Base.error(error, this.$root, error.message);
+        })
+        .finally(() => {
+          this.isBusy = false;
+        });
+    },
+
+    /**
+     * Handle update settings data
+     *
+     * @param settings Settings object to fill the payload
+     */
+    updateSettings (settings) {
+      this.isBusy = true;
+
+      // Build form data
+      const formData = new FormData();
+      if (this.uploadLogoFile) {
+        formData.append('logo_file', this.uploadLogoFile);
+      } else {
+        formData.append('logo', settings.logo);
+      }
+      if (this.uploadFaviconFile) {
+        formData.append('favicon_file', this.uploadFaviconFile);
+      } else {
+        formData.append('favicon', settings.favicon);
+      }
+      formData.append('name', settings.name);
+      formData.append('room_limit', settings.roomLimit);
+      formData.append('pagination_page_size', settings.paginationPageSize);
+      formData.append('own_rooms_pagination_page_size', settings.ownRoomsPaginationPageSize);
+      formData.append('_method', 'PUT');
+
+      Base.call('settings',
+        {
+          method: 'post',
+          data: formData,
+          headers: {
+            'Content-Type': 'multipart/form-data'
+          }
+        })
+        .then(response => {
+          this.$store.dispatch('session/getSettings');
+          this.errors = {};
+          this.uploadLogoFile = null;
+
+          // update form input
+          this.settings.logo = response.data.data.logo;
+          this.settings.favicon = response.data.data.favicon;
+          this.settings.name = response.data.data.name;
+          this.settings.roomLimit = response.data.data.room_limit;
+          this.settings.ownRoomsPaginationPageSize = response.data.data.own_rooms_pagination_page_size;
+          this.settings.paginationPageSize = response.data.data.pagination_page_size;
+          this.roomLimitMode = (this.settings.roomLimit === -1 ? 'unlimited' : 'custom');
+        })
+        .catch((error) => {
+          if (error.response) {
```

Comment:
this can be just 
```suggestion
          if (error.response && error.response.status === env.HTTP_UNPROCESSABLE_ENTITY) {
            this.errors = error.response.data.errors;
          } else {
            Base.error(error, this.$root);
          }
```